lib/Driver/ToolChains/Flang.cpp

```diff
@@ -772,12 +774,13 @@ void FlangFrontend::ConstructJob(Compilation &C, const JobAction &JA,
   UpperCmdArgs.push_back("-modindex");
   UpperCmdArgs.push_back(ModuleIndexFile);
 
-  UpperCmdArgs.push_back("-output");
```

Comment:
If `-o` is not given, then `-output` should not be sent to flang1, right?

Reply:
It's dependent on context. If run in `-E` mode, `-o` will output the preprocessed file. If run in regular mode, `flang1` will output the ILM file.

Reply:
I mean if `-E` is given and `-o` is not given, it should print to stdout which means we shouldn't send `-output` to flang1